xml/security_preface.xml

```diff
@@ -8,753 +8,450 @@
-     become vulnerable to <emphasis>man-in-the-middle attacks</emphasis>
-     (sniffing, TCP connection hijacking, spoofing) and DNS poisoning.
-    </para>
-   </sect3>
-   <sect3 xml:id="sec.security.general.overview.network.mitm">
-    <title>Man in the Middle: Sniffing, Hijacking, Spoofing</title>
-    <para>
-     In general, any remote attack performed by an attacker who puts himself
-     between the communicating hosts is called a <emphasis>man-in-the-middle
-     attack</emphasis>. What almost all types of man-in-the-middle attacks
-     have in common is that the victim is usually not aware that there is
-     something happening. There are many variants. For example, the
-     attacker could pick up a connection request and forward that to the
-     target machine. Now the victim has unwittingly established a connection
-     with the wrong host, because the other end is posing as the legitimate
-     destination machine.
-    </para>
-    <para>
-     The simplest form of a man-in-the-middle attack is called
-     <emphasis>sniffer</emphasis> (the attacker is <quote>only</quote>
-     listening to the network traffic passing by). As a more complex attack,
-     the <quote>man in the middle</quote> could try to take over an already
-     established connection (hijacking). To do so, the attacker would need
-     to analyze the packets for some time to be able to predict the TCP
-     sequence numbers belonging to the connection. When the attacker finally
-     seizes the role of the target host, the victims notice this, because
-     they get an error message saying the connection was terminated because
-     of a failure. That there are protocols not secured against
-     hijacking through encryption (which only perform a simple
-     authentication procedure upon establishing the connection) makes it
-     easier for attackers.
-    </para>
+ <sect1 xml:id="sec.security.network">
+  <title>Networking</title>
+  <para>
+   Securing network services is a crucial task. It is useful to secure
+   as many layers of the <emphasis>OSI model</emphasis> as possible.
+  </para>
+  <para>
+   All communication should be authenticated and encrypted with
+   up-to-date cryptographic algorithms on the transport or application
+   layer. Use a Virtual Private Network (VPN) as an additional secure
+   layer on physical networks.
+  </para>
+  <para>
+   &productname; provides many options for securing your network:
+  </para>
+  <itemizedlist>
+   <listitem>
     <para>
-     <emphasis>Spoofing</emphasis> is an attack where packets are modified
-     to contain counterfeit source data, usually the IP address. Most active
-     forms of attack rely on sending out such fake packets (something that,
-     on a Linux machine, can only be done by the superuser
-     (<systemitem class="username">root</systemitem>)).
+     Use &yast; to create X509 certificates. These certificates can be
```

Comment:
fyi, the `yast-ca-management` is (temporarily, i.e. at least until 15 SP1) unavailable, FATE#319119.